cypress/integration/accessibility_IconButton_spec.js

```diff
@@ -5,6 +5,15 @@ describe('IconButton Accessibility check', () => {
   });
 
   it('Tests accessibility on the IconButton page', () => {
+    cy.configureAxe({
+      rules: [
+        {
+          id: 'button-name',
```

Comment:
Curios why this needed to be added

Reply:
Also feel free to disable color contrast, not sure why that's failing, it's confused about the background color behind the "Tooltip" link

Reply:
DONE

Reply:
@ayeshakmaz With tooltip, to prevent redundancy with accessibility label, IconButton as empty "" label, and it complains about that.

Reply:
Ahhh that's right.. sounds good